pkg/sessions/session_store_test.go

```diff
@@ -239,6 +277,28 @@ var _ = Describe("NewSessionStore", func() {
 		})
 	})
 
+	Context("with type 'redis'", func() {
+		BeforeEach(func() {
+			mr, err := miniredis.Run()
```

Comment:
We should probably add an `AfterEach` to tear down the miniredises that we are spawning right? Otherwise we are gonna end up with lots of redises 😆